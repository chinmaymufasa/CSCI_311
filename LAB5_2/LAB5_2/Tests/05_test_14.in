2
53
4
5
6
2
77
4
5
6
2
39
4
5
6
2
18
4
5
6
2
170
4
5
6
-1
