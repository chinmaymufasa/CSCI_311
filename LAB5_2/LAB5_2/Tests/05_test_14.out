53 
53 
53 
53 77 
53 77 
77 53 
53 39 77 
39 53 77 
39 77 53 
53 39 18 77 
18 39 53 77 
18 39 77 53 
53 39 18 77 170 
18 39 53 77 170 
18 39 170 77 53 
